src/libs/OptionsListUtils.js

```diff
@@ -136,15 +178,35 @@ function createOption(personalDetailList, report, draftComments, {showChatPrevie
             : '')
         + _.unescape(report.lastMessageText)
         : '';
+
+    const isDefaultChatRoom = isDefaultRoom(report);
+    const policyInfo = policies[lodashGet(report, ['policyID'], '')];
     const tooltipText = getReportParticipantsTitle(lodashGet(report, ['participants'], []));
-    const fullTitle = personalDetailList
-        .map(({firstName, login}) => firstName || Str.removeSMSDomain(login))
-        .join(', ');
-    return {
-        text: hasMultipleParticipants ? fullTitle : report?.reportName || personalDetail.displayName,
```

Comment:
I changed this line to 
```
lodashGet(report, ['reportName'], personalDetail.displayName)
```
since our style doc says to avoid optional chaining. It has the same logic, but I just wanted to note why I changed it.